internal/api/controllers/private/runsCreate.go

```diff
@@ -12,10 +12,13 @@ import (
 
 	"github.com/google/uuid"
 	"github.com/labstack/echo/v4"
+	"go.uber.org/ratelimit"
 )
 
 var cfg = config.Get()
 
+var RateLimiter = ratelimit.New(cfg.GetInt("cloud.connector.rps"))
```

Comment:
Breaking the component encapsulation like this (i.e. making this field public although it's only used internally) is not great. It would be better to keep it private and set the config setting e.g. in https://github.com/RedHatInsights/playbook-dispatcher/blob/master/internal/api/tests/private/private_suite_test.go#L40